### ISSUE TITLE
Documentation update for richtext editor, according to Volto 15

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#3075]
https://docs.voltocms.com/configuration/richeditor-settings/ needs to be updated for new configuration method.

**Comment by punithnayak** [comment]
@tiberiuichim @stevepiercy Hey,,,i will do this but can you please specify what exactly should I be doing .I am a newbie to open-source.Can you provide we with resources to solve the issue. Thank you

**Comment by stevepiercy** [comment]
@punithnayak start with reading the documentation and how to contribute.

**Comment by tiberiuichim** [comment]
@punithnayak This ticket is about the older implementation of Volto's richtext editor, based on draftjs. At one point we've changed the way the configuration needs to be declared.

There are plans to remove draftjs from Volto, just like we did with sentry. The problem with the current documentation (if we want to update it) is that it needs to:

- explain that Draftjs is no longer the default (it's `@plone/volto-slate`)
- eventually needs to be update to keep up with the API change

As far as I can say, a better effort would be spent to move draftjs to a new `@plone-collective/volto-draftjs` addon.

@punithnayak Take a look at the issues in this project, https://github.com/plone/volto/projects/35 there are a few of them that are fit to be used as introductory to the whole stack and contributing to Volto.
